components/actionbar/index.css

```diff
@@ -168,6 +136,20 @@
 	.spectrum-FieldLabel {
 		color: var(--mod-actionbar-emphasized-item-counter-color, var(--spectrum-actionbar-emphasized-item-counter-color));
 	}
+
+	.spectrum-ActionGroup .spectrum-ActionButton {
+		.spectrum-ActionButton-label {
+			--mod-actionbutton-label-color: var(--spectrum-actionbar-emphasized-action-button-label-color);
```

Comment:
I wonder if, for the emphasized action bar specifically in dark mode in force colors/WHCM 🥵 if this should use a high contrast variable. Testing it in both the Chrome forced-colors emulator and Assistiv labs, if I switch to dark mode, the emphasized action buttons are not visible. 
<img width="697" alt="Screenshot 2025-04-23 at 5 26 34 PM" src="https://github.com/user-attachments/assets/47154eff-598b-4d3f-9ace-944dad3e4c34" />

```
--mod-actionbutton-label-color: var(--highcontrast-actionbar-emphasized-action-button-label-color, var(--spectrum-actionbar-emphasized-action-button-label-color));
```

I don't know where quite yet we'd want that mod, if it's here or somewhere else.

Reply:
Oh, and somewhat related- I'd like to see this action button mod passthrough be organized with the rest of the custom property definitions, instead of mixed in with the style definitions.